FinalProject/database_setup.py

```diff
@@ -4,20 +4,65 @@
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm import relationship
 from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from flask import Flask
+from flask.ext.sqlalchemy import SQLAlchemy
```

Comment:
We are not using Flask extensions because it better for the student to learn SQLAlchemy API's directly rather than using it through a wrapper. Later on if they find it useful, they are welcome to use it but for the course it is not needed.